ffs/integrationtest/general/general_test.go

```diff
@@ -105,12 +106,45 @@ func TestInfo(t *testing.T) {
 
 		r := rand.New(rand.NewSource(22))
 		n := 2
+		retriesPerDeal := 5
 		for i := 0; i < n; i++ {
-			cid, _ := it.AddRandomFile(t, r, ipfs)
-			jid, err := fapi.PushStorageConfig(cid)
-			require.NoError(t, err)
-			it.RequireJobState(t, fapi, jid, ffs.Success)
-			it.RequireStorageConfig(t, fapi, cid, nil)
+			for j := 0; j < retriesPerDeal; j++ {
```

Comment:
So `TestInfo` is mostly the only remaining flaky test. The `RunFlaky(..)` trick doesn't quite work here for the following reason: This test make two deals per test, so if any of both deals fails then a retry should be made. But the probability of failure is much bigger than other tests, since if _any_ of those fail that would be enough for failing.

This might remediate that... which tries to have some extra attempts "per retry" in each deal that should be made. A bit of a stretch. I lost quite a lot of time making this pass, so a bit tired of its behaviour.

If still gets flaky, we should make `n := 1` and adjust the test. It wouldn't be totally ideal from my POV, but it would check the important thing of the test anyway. I'm very frustrated about this local-devnet flakyness.